lib/gosu/image.rb

```diff
@@ -56,42 +64,42 @@ def self.load_tiles(filename_or_image, tile_width ,tile_height, retro: false, ti
       callback = proc { |data, image| images << Gosu::Image.new(image, retro: retro, tileable: tileable) }
 
       if filename_or_image.is_a?(String)
-        _image_load_tiles(filename_or_image, tile_width, tile_height, callback, nil, flags)
+        Gosu_Image_load_tiles(filename_or_image, tile_width, tile_height, callback, nil, flags)
       else
         if filename_or_image.instance_of?(Gosu::Image)
-          _image_load_tiles_from_image(filename_or_image.__pointer, tile_width, tile_height, callback, nil, flags)
+          Gosu_Image_load_tiles_from_image(filename_or_image.__pointer, tile_width, tile_height, callback, nil, flags)
```

Comment:
`Gosu_Image_load_tiles_from_image` should be `Gosu_Image_create_tiles_from_image`